### DISCUSSION TITLE
Installation Help

### DISCUSSION THREAD

**Discussion by anubhav06** [osm-search/Nominatim#2631] [Q&A] [answered]
Hey, I was following the installation instructions for [Ubuntu 20.04](https://nominatim.org/release-docs/develop/appendix/Install-on-Ubuntu-20/)
  
  
![Screenshot 2022-03-14 215527](https://user-images.githubusercontent.com/43821031/158216669-bbad892b-8aed-42a1-bca6-bcb85ab51558.png)

  
  

I am fairly new with Linux, so do I have to switch my user to `nominatim` after it says `Never run the installation as root user`

If yes, then how so ?
I know that `su - <user>` switches the user, and it asks for the password, but I was not asked to create any password for the user when I ran the command mentioned in docs :`sudo useradd -d /srv/nominatim -s /bin/bash -m nominatim`

Any help is appreciated. Thanks!

**Comment by jspinella** [comment, marked as answer]
I don't see why not, but you can also use your user e.g. "agupta". None of the commands are user-specific because we are using the variable USERNAME to run commands for the "nominatim" user. 

E: unless the nominatim user isn't a member of the sudoers (in which case it can't execute commands as root (sudo) and you'd get an error). Try it, if it doesn't work, use your personal user account.

**Reply by anubhav06** [reply]
Got it! Thanks a lot for answering!! 😄

**Reply by jspinella** [reply]
And to be super clear, I would not add the nominatim user to sudoers if it isn't by default. I can't recall if it would be from `sudo useradd` but I think not. You usually don't want linux users that are being used solely for services (e.g. the postgres user for PostgreSQL) to have the ability to sudo commands.
